amplipi/streams.py

```diff
@@ -511,7 +511,7 @@ def disconnect(self):
     self.src = None
 
   def info(self):
-    loc = '/home/pi/config/dlna/{}/currentSong'.format(self.src)
+    loc = '/home/pi/config/srcs/{}/currentSong'.format(self.src)
```

Comment:
Can you rebase this PR on some of the recent restructuring to get utils.get_dir('config') and use that to remove the absolute paths?